buildSrc/src/main/groovy/com/uber/okbuck/core/model/android/AndroidTarget.groovy

```diff
@@ -141,14 +142,20 @@ abstract class AndroidTarget extends JavaLibTarget {
 
     @Override
     Scope getApt() {
-        return Scope.from(project, baseVariant.annotationProcessorConfiguration)
+        if (!isKotlin)
+            return Scope.from(project, baseVariant.annotationProcessorConfiguration)
+        else
+            return Scope.from(project, KotlinLibTarget.ANNOTATION_PROCESSOR_CONFIGURATION_NAME)
```

Comment:
This seems like a bad code smell. Android target depending on a constant from kotlin target.

Lets move constant outside or use the ones defined by the kotlin gradle plugin api directly

Reply:
This should actually probably use the `baseVariant`, but I honestly don't know how `annotationProcessorConfiguration` is built